.travis.yml

```diff
@@ -24,3 +24,6 @@ script:
     - make lint
     - make test
     - make coverage
+
+after_success:
+    - bash <(curl -s https://codecov.io/bash)
```

Comment:
Can you put a comment on this to indicate that this is how codecov.io prefers reports be uploaded?  or a link to their documentation page or something?  Thanks!

Reply:
Done.